### DISCUSSION TITLE
Extended access has ended

### DISCUSSION THREAD

**Discussion by hassaanp** [community/community#44196] [Copilot Conversations] [answered]
### Select Topic Area

Question

### Body

I am seeing this prompt in the billing section

`Thank you for participating in the GitHub Copilot Technical Preview. Your extended access has ended.` 

See screenshot
<img width="786" alt="Screenshot 2023-01-13 at 2 21 15 PM" src="https://user-images.githubusercontent.com/10243572/212286414-71a26d9d-dfb7-4ee2-b533-75c5fc979f35.png">

What does this mean? I can still use copilot on vscode but what does extended access refer to?

**Comment by radar-blip** [comment, marked as answer]
According to this post #43634 lots of people are having same issue with copilot.
See selected answer.

**Reply by hassaanp** [reply]
Thank you for the quick response.
